src/main/src/main/java/org/geoserver/security/decorators/DecoratingWMTSLayerInfo.java

```diff
@@ -0,0 +1,235 @@
+/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
```

Comment:
For new files the copyright should be:

````
/* (c) 2017 Open Source Geospatial Foundation - all rights reserved
 * This code is licensed under the GPL 2.0 license, available at the root
 * application directory.
 */
````

Please fix it on all new files. Existing files copyright headers do not need touching